lib/fileHandler/index.js

```diff
@@ -47,6 +47,20 @@ const DEFAULT_UPLOAD_OPTIONS = {
 	queueSize: 4 //Ensures at least up to 1/4 of the RAM is utilised
 };
 
+const REQUIRED_FILE_PROPS = ['name', 'url', 'mime_type', 'expires']
+
+const isObjValid = fileObj => {
+    const fileObjKeyArr = _.keys(fileObj)
+    return REQUIRED_FILE_PROPS.every(reqField => fileObjKeyArr.includes(reqField))
```

Comment:
It would be good if the validation also checked the types of each value, so name, url and mime type should be strings, while expires is an integer.